mlir-tensorrt/executor/include/mlir-executor-c/Runtime/Runtime.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
 /// caller must be sure to delete errors via mtrtStatusDestroy.
 //===----------------------------------------------------------------------===//
 
-typedef struct MTRT_RuntimeClient MTRT_Runtimeclient;
+struct MTRT_RuntimeClient; // Forward declaration
```

Comment:
This is a C header, not C++. 

Without the typedef, we would fail to compile when including this header in a C library (e.g. we would need to rename all the types below to `struct MTRT_RuntimeClient` instead of just `MTRT_RuntimeClient`. The `typedef` is a requirement.

Reply:
any idea why we use `MTRT_Runtimeclient` instead of `MTRT_RuntimeClient`? Is it just a typo?